app/src/main/java/com/example/poste/activities/EditPostActivity.java

```diff
@@ -113,6 +112,7 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
                             currentPost.setDescription(description);
                             currentPost.setUrl(url);
                             Toast.makeText(EditPostActivity.this,"Edit post successful!", Toast.LENGTH_SHORT).show();
+                            finish();
```

Comment:
Would suggest placing this after the if/else for response.isSuccessful, that way it returns on either case; onFailure as well